spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -256,23 +247,13 @@
       end
 
       after(:each) do
-        RepoAccess.destroy_all
-        Group.destroy_all
         GroupAssignmentRepo.destroy_all
       end
 
-      it "redeems the users invitation" do
-        patch :accept_invitation, params: { id: invitation.key, group: { title: "Code Squad" } }
-
-        expect(WebMock).to have_requested(:post, github_url("/organizations/#{organization.github_id}/teams"))
-        expect(WebMock).to have_requested(:post, github_url("/organizations/#{organization.github_id}/repos"))
-
-        expect(group_assignment.group_assignment_repos.count).to eql(1)
-        expect(student.repo_accesses.count).to eql(1)
-      end
```

Comment:
Removed this test since it's actually testing some dead code (I need to get around to removing a bunch of dead code).